examples/pixelcnn/model_test.py

```diff
@@ -0,0 +1,129 @@
+# Lint as: python3
+"""Tests for model."""
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import numpy.testing as onp_testing
+
+from jax import random
+import jax.numpy as np
+
+from flax import nn
+
+import pixelcnn
+
+
+class ModelTest(absltest.TestCase):
```

Comment:
Super nice!

Reply:
Given all these masked convolutions, would be awesome to also include a test to check the receptive field of the model. 
Similar to what we have here: https://github.com/google/flax/blob/master/tests/nn_attention_test.py#L146